src/Raven.Server/Rachis/Follower.cs

```diff
@@ -425,9 +492,9 @@ private void NegotiateWithLeader(TransactionOperationContext context, LogLengthN
                     }
 
                     //This is okay to ignore because we will just get the committed entries again and skip them
-                    ReadInstallSnapshotAndIgnoreContent(context);
+                    ReadInstallSnapshotAndIgnoreContent();
```

Comment:
Reading the content may take a while (2GB, frex). I think we ought to provide the `cts` here as well

Reply:
done